packages/smart/addresses.ts

```diff
@@ -323,79 +221,62 @@ export const addresses: AddressMapping = {
       {
         type: "MLB",
         subtype: "V3",
-        address: "0xB4F6EFd8aD6ad409b032B44F12F7a9caA6B9286A",
+        address: "0xEF7381F25405dDda78c8745ff8Fc59C9dC380408",
         collateral: "0x5799bFe361BEea69f808328FF4884DF92f1f66f0",
-        ammFactory: "0x3eaB11C23b6Df47BE57576D9bB929aeCF54CF5Ee",
-        fetcher: "0x5a556A55E23678f1700Dd22A2f05aa0BF9313534",
+        ammFactory: "0x27F6291ffdc1D8701F4243837596633B35390Db5",
+        fetcher: "0x6F9F76a5B4CD7B2c497291c938d284D3b5704232",
         hasRewards: true,
+        masterChef: "0x47756f374B331B5afDD445276119929c8db61d28",
         description: "mlb",
-        version: "renaming-2021-09-08",
+        version: "FILL THIS OUT",
```

Comment:
I assume these are still waiting on some decision but noting anyway since it is not done

Reply:
Yep, that relates to the release.